src/backend/rendering/ctextrendering.cpp

```diff
@@ -305,24 +305,8 @@ QString CTextRendering::renderEntry(KeyTreeItem const & i, CSwordKey * k)
                 QString(" xml:lang=\"%1\" lang=\"%1\"").arg(
                     modulePtr->language()->abbrev());
 
-        if (key->isValid() && i.key() == key->key()) {
```

Comment:
Here too, not sure why we compare current key with a key to render, but it may follow through i think.

Reply:
This should not be here